### PR TITLE
Make use of the new contributor feature.

### PR DESCRIPTION
**Pull request by LaxWasHere**
Implements contributors

### CODE REVIEW

File: Essentials/src/plugin.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ website: http://tiny.cc/EssentialsCommands
 description: Provides an essential, core set of commands for Bukkit.
 softdepend: [Vault, LuckPerms]
 authors: [Zenexer, ementalo, Aelux, Brettflan, KimKandor, snowleo, ceulemans, Xeology, KHobbits, md_5, Iaccidentally, drtshock, vemacs, SupaHam, md678685]
+contributors: [LaxWasHere, pop4959, JRoy, Triagonal, zml2008]
```
Comment (mdcfe) [inline Essentials/src/plugin.yml:10]:
```suggestion
authors: [Zenexer, ementalo, Aelux, Brettflan, KimKandor, snowleo, ceulemans, Xeology, KHobbits, md_5, Iaccidentally, drtshock, vemacs, SupaHam, md678685, pop4959, JRoy, Triagonal, zml2008]
```

**Review by mdcfe — CHANGES_REQUESTED** [review]

File: Essentials/src/plugin.yml (hunk 1 above)
Comment (LaxWasHere) [reply Essentials/src/plugin.yml:10]: > 

Does not compile using java 5.

Comment (mdcfe) [reply Essentials/src/plugin.yml:10]: no u

Comment (LaxWasHere) [reply Essentials/src/plugin.yml:10]: w0w

Comment (JRoy) [inline Essentials/src/plugin.yml:10]:
```suggestion
authors: [md_5, ]
contributors: [does, not, matter, 2008Chooco]
```

**Review by JRoy — CHANGES_REQUESTED** [review]

File: Essentials/src/plugin.yml (hunk 1 above)
Comment (mdcfe) [reply Essentials/src/plugin.yml:10]: Wow that's controversial

Comment (JRoy) [reply Essentials/src/plugin.yml:10]: Added 2008Chooco to the contributors list for making the contributors list

Comment (mdcfe) [reply Essentials/src/plugin.yml:10]: `2008Choco` is not a contributor to EssentialsX

Comment (JRoy) [reply Essentials/src/plugin.yml:10]: he invented contributors

Comment (mdcfe) [reply Essentials/src/plugin.yml:10]: no u

Comment (Draycia) [reply Essentials/src/plugin.yml:10]:
```suggestion
authors: [how, do, i, exit, vim]
```

**Comment by pop4959** [comment]
Closed because the contributors field is not compatible with all versions of MineCraft that EssentialsX supports. If you can find a way to make this work on 1.8.8-1.16.1 feel free to open another PR.

**Pull request closed without merging**